models/ZFIN_ZDB-GENE-010102-1.ttl

```diff
@@ -0,0 +1,391 @@
+@prefix ECO: <http://purl.obolibrary.org/obo/ECO_> .
+@prefix GO: <http://purl.obolibrary.org/obo/GO_> .
+@prefix ZFIN: <http://identifiers.org/zfin/> .
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix ns1: <http://geneontology.org/lego/> .
+@prefix ns2: <http://purl.org/pav/> .
+@prefix obo: <http://purl.obolibrary.org/obo/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+
+ns1:evidence a owl:AnnotationProperty .
+
+<http://geneontology.org/lego/hint/layout/x> a owl:AnnotationProperty .
+
+<http://geneontology.org/lego/hint/layout/y> a owl:AnnotationProperty .
+
+dc:contributor a owl:AnnotationProperty .
+
+dc:date a owl:AnnotationProperty .
+
+dc:source a owl:AnnotationProperty .
+
+ns2:providedBy a owl:AnnotationProperty .
+
+<http://model.geneontology.org/ZFIN_ZDB-GENE-010102-1> a owl:Ontology ;
+    ns1:modelstate "development" ;
+    dc:contributor "http://orcid.org/0000-0002-6659-0416" ;
+    dc:date "2021-3-1" ;
+    dc:title "ZFIN:ZDB-GENE-010102-1" ;
+    ns2:providedBy "http://geneontology.org" ;
+    owl:versionIRI <http://model.geneontology.org/ZFIN_ZDB-GENE-010102-1> .
+
+<http://model.geneontology.org/ZFIN_ZDB-GENE-010102-1/0ebbb92d-a1de-46c5-8656-f3896e346349> a ECO:0000315,
+        owl:NamedIndividual ;
+    ns1:evidence-with "ZFIN:ZDB-MRPHLNO-090513-6" ;
+    dc:contributor ":",
+        "C",
+        "G",
+        "O",
+        "_",
+        "a",
+        "c",
+        "f",
+        "i",
+        "n",
+        "o",
+        "r",
+        "s",
+        "t",
+        "u",
+        "z" ;
+    dc:date "2009-05-14" ;
```

Comment:
@sierra-moxon I see the weird split contributor bug has returned. Is this using the latest version of ontobio?

Reply:
@dustine32 - good catch - fixing.